config/manager/kustomization.yaml

```diff
@@ -12,4 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/mdewald/managed-node-metadata-operator
```

Comment:
this is unfortunately done each time you run a make target... can't remember which, or the app-sre pipeline scripts. Should be (and stay) `app-sre`